### PR TITLE
Update variable names and language used

### PR DESCRIPTION
**Pull request by mcdonnnj**
#  #

## 🗣 Description ##

This pull request renames some variables, adjusts their descriptions, renames some facts, and adjusts some task names.

## 💭 Motivation and context ##

While working on #48 I noticed some inconsistencies and/or not quite accurate information for some variables, facts, and task names. This pull request simply seeks to clean that up and better document the functionality in this role.

## 🧪 Testing ##

Automated tests pass.

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All relevant repo and/or project documentation has been updated
      to reflect the changes in this PR.
- [x] All new and existing tests pass.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,8 +63,8 @@ None.
 
 | Variable | Description | Default | Required |
 |----------|-------------|---------|----------|
-| geoip2\_local\_file | The filename used to store the downloaded database. | `GeoIP2-City.tar.gz` | No |
-| geoip2\_local\_path | The directory to extract the database into. | `/usr/local/share/GeoIP/` | No |
+| geoip2\_archive\_filename | The filename to use for the downloaded database archive. | `GeoIP2-City.tar.gz` | No |
+| geoip2\_database\_directory | The directory to store the database files. | `/usr/local/share/GeoIP/` | No |
```
Comment (jsf9k) [inline README.md]:
```suggestion
| geoip2\_database\_directory | The directory in which to store the database files. | `/usr/local/share/GeoIP/` | No |
```

File: defaults/main.yml [hunk 2]
```diff
@@ -1,7 +1,9 @@
 ---
+# Name to use for the downloaded database archive
+geoip2_archive_filename: GeoIP2-City.tar.gz
+
 # Directory to store GeoIP files
```
Comment (jsf9k) [inline defaults/main.yml]:
```suggestion
# Directory in which to store GeoIP files
```

File: tasks/main.yml [hunk 3]
```diff
@@ -45,47 +45,47 @@
...
+          checksum_url,
           username=geoip2_maxmind_account_id,
           password=geoip2_maxmind_license_key
         )
         | split
         | first
       }}
 
-- name: Create the local directory
+- name: Ensure the directory to store the database file exists
```
Comment (jsf9k) [inline tasks/main.yml]:
```suggestion
- name: Ensure the directory in which to store the database file exists
```

**Review by jsf9k — APPROVED** [review]
I had a few suggestions to increase readability.

**Review by dv4harr10 — APPROVED** [review]

**Pull request merged**